docs/Serverless.md

```diff
@@ -1,5 +1,8 @@
 <h1 align="center">Serverless</h1>
 
+> Fastify is not designed to run on the AWS Lambda platform. The Fastify framework is designed to make implementing a traditional HTTP/S server easy. The AWS Lambda platform services requests in a completely different manner than a traditional HTTP/S server and is therefore at odds with Fastify's whole design. Regardless, it is possible to use Fastify in this environment if you so choose. This document describes how to do it. However, remember that this is an unsupported usage of Fastify and you are on your own in terms of support.
```

Comment:
I would actually states "serverless environments" instead of "AWS Lambda" in this.

Also, can you please word-wrap this at around ~80 chars?

Reply:
√

Reply:
He means to add literal line breaks in the document at ~80 characters. He does not mean to insert `<br>` elements.

Reply:
like this? https://github.com/fastify/fastify/pull/1564/commits/88c6c6a8389aa8905aacec948c3ee7b60b8bc2ea

Reply:
Yes. The whole document should follow the same guideline.

Reply:
✅